.env_example

```diff
@@ -0,0 +1,3 @@
+GITHUB_TOKEN='YOUR_GITHUB_TOKEN_HERE'
```

Comment:
To clarify, do we need our tokens to be enclosed in quotes? because we don't have them like that currently. If not, we should probably remove those quotes

Reply:
It was just the style I saw in documentation about putting things in the .env file. I know when I've saved keys from the shell command line I've had to use quotes to process them as strings.

https://unix.stackexchange.com/questions/16303/what-is-the-significance-of-single-and-double-quotes-in-environment-variables

I guess this article points out a problem with docker, so that's enough reason for me:
https://dev.to/tvanantwerp/don-t-quote-environment-variables-in-docker-268h